README.md

```diff
@@ -92,19 +92,52 @@ The R53 domains for non-production and production are stored in different AWS ac
 
 example `extensions.yml` config.
 
+```yaml
+my-application-alb:
+  type: alb
+  environments:
+    dev: 
+      additional_address_list:
+        - internal.my-web-service-2
+```
+
+## CDN
+
+This module will create the CloudFront (CDN) endpoints for the application if enabled.
+
+Additional domains (`cdn_domains_list`) are the domain names that will be configured in CloudFront. In the map the key is the fully qualified domain name and the value is the application's base domain (the application's Route 53 zone).  
```

Comment:
It's confusing to us "Additional domains" here.

Also, the instruction on the value appears to be wrong based on the information under the example. Can the be lifted up here and removed from below?
```suggestion
`cdn_domains_list` is a map of the domain names that will be configured in CloudFront.
* the key is the fully qualified domain name
* the value is an array containing the internal prefix and the base domain (the application's Route 53 zone).  
```